### ISSUE TITLE
1100+ datasets with fossils/extinct taxa that need to be flagged as such to be merged on COL

### ISSUE THREAD

**Issue by camiplata** [plazi/community#348]
Currently fossils/extinct taxa enters COL as extant taxa wich can be misleading for users. I checked the PALZI datasets that are been merged into COL via the extended release and look for Paleo keywords, such as Fossil or geological periods. The following 800+ datasets are very like to need an update to flagged the taxa as extinct. [Plazi_PaleoTaxa.csv](https://github.com/user-attachments/files/17824463/Plazi_PaleoTaxa.csv)

How: In DwC speciesProfile extension, element extinct=true, or in COLDP nameusage, extict= true

@mdoering do you have further recommendations, do we have a better way to sort fossil vs. extinct?

**Comment by mdoering** [comment]
That's the most common way to do it. Alternatively the name can be preceeded with the dagger symbol:
>†Aspredinidae

In addition you can provide the geological time range the species is know to have lived from the known fossil record.
For ColDP: [Taxon.temporalRangeStart](https://github.com/CatalogueOfLife/coldp/blob/master/README.md#temporalrangestart) & end
For DwC species profile [gbif:livingPeriod](https://rs.gbif.org/extension/gbif/1.0/speciesprofile_2019-01-29.xml)

**Comment by camiplata** [comment]
I did an update on this list, now it has 1129 datasets, and includes the Plazi ID an URL

[PaleoSectors_January2025.csv](https://github.com/user-attachments/files/18363410/PaleoSectors_January2025.csv)

**Comment by mdoering** [comment]
I have just tried the extinct dataset import setting in CLB and it works, but then flags all taxa incl higher ones like Animalia as extinct. So that's a bit useless. I will change the setting to accept a max rank instead and only flag names equals to and below that rank.

@camiplata would you think all genera and species below should be fossils in your dataset list?

**Comment by camiplata** [comment]
Yes Markus, that will work for most of the cases.  Few of them have fossil tribes, but we can leave the classification  for PBDB

**Comment by camiplata** [comment]
@flsimoes Have you consider flagging extinct taxa from the treatment already?

**Comment by flsimoes** [comment]
> [@flsimoes](https://github.com/flsimoes) Have you consider flagging extinct taxa from the treatment already?

Sorry for the delay @camiplata 
I honestly don't know the inner workings of how this sort of flagging could be done.
I need to confer with Guido and/or Markus for feasibility, but my guess is that it would be possible (provided there's an appropriate mapping to the dwca).

**Comment by camiplata** [comment]
@flsimoes Did you have chance to discuss tho matter with Guido?
